Engines/Wine/Engine/Implementation/script.js

```diff
@@ -447,9 +450,9 @@ module.default = class WineEngine {
         if (architecture == "amd64") {
             ldPath =
                 this._wineEnginesDirectory +
-                "/runtime/lib64/:" +
+                (this._useRuntime ? "/runtime/lib64/:" : "") +
```

Comment:
I think an `if (this._useRuntime) { ... } else { ... }` may be easier to read and understand

Reply:
It is perfectly readable. There is not a lot of ternary operator ^^.

Reply:
If I use `if else`, I cannot do it inline. So I would have to repeat much of the code which is not readable either.

Reply:
Actually I'm against an inline solution. I think a single string would be the most readable, e.g.:
```javascript
ldPath = `${this._wineEnginesDirectory}/runtime/lib/:/runtime/lib/:${this.getLocalDirectory(subCategory, version)}/lib/:${ldPath}`;
```

Reply:
An alternative would be to define the parts of `ldPath` in separate variables and concat them at the end:

```javscript
ldPath = `${phoenicisRuntime}:${systemRuntime}:${localLibraries}:${ldPath}`;
```

Reply:
Better?